<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPersonEdit</class>
 <widget class="QDialog" name="DialogPersonEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="verticalSpacing">
      <number>12</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="l_dsurname">
       <property name="text">
        <string>Фамилия</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_dsurname"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="l_dname">
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_dname"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="l_dpatronymic">
       <property name="text">
        <string>Отчество</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_dpatronymic"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="l_dage">
       <property name="text">
        <string>Возраст</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="le_dage"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="l_dtelephone">
       <property name="text">
        <string>Телефон</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pb_dsave">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="le_dphone"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
